frontend/app/package.json

```diff
@@ -58,6 +58,7 @@
     "eslint-plugin-vue": "9.17.0",
     "eslint-plugin-vuetify": "1.1.0",
     "imask": "6.6.2",
+    "json-editor-vue": "0.10.19",
```

Comment:
do you think it would be too much effort to port this to our codebase and use `vanilla-jsoneditor` directly? It seems that the whole library is a single class wrapper around `vanilla-jsoneditor` either way.